server/src/services/library.service.spec.ts

```diff
@@ -337,12 +337,35 @@ describe(LibraryService.name, () => {
 
       expect(assetMock.updateAll).toHaveBeenCalledWith([assetStub.trashedOffline.id], {
         deletedAt: null,
-        fileCreatedAt: assetStub.trashedOffline.fileModifiedAt,
         fileModifiedAt: assetStub.trashedOffline.fileModifiedAt,
         isOffline: false,
         originalFileName: 'path.jpg',
       });
     });
+
+    // regression test for https://github.com/immich-app/immich/issues/15640
+    it('should not touch fileCreatedAt when un-trashing an asset previously marked as offline', async () => {
```

Comment:
I think this deception for the test is sufficient. We didn't need to link to GitHub issues or have other comments here.

Reply:
So you'd rather have both comments (above and inside) removed?

Reply:
I agree with Jason and yes, I'd remove both comments. I think the name of the test is quite descriptive.

Reply:
Alright, done